python/cudf/cudf/utils/ioutils.py

```diff
@@ -1319,7 +1319,7 @@ def _open_remote_files(
     ]
 
 
-def get_filepath_or_buffer(
+def get_reader_filepath_or_buffer(
```

Comment:
I'm not sure why we needed to rename this function. The word `reader` doesn't add any meaning to me.

Reply:
I had to name it `get_reader_filepath_or_buffer` because we have a `get_writer_filepath_or_buffer`. Both used by reader and writer APIs respectively.

Reply:
That's a good decision, then. I looked at this file recently but hadn't noticed `get_writer_filepath_or_buffer`. 😄